### ISSUE TITLE
Missing package versions for DataFrames during installation

### ISSUE THREAD

**Issue by alvations** [johnmyleswhite/TextAnalysis.jl#52]
From a fresh installation of julia on Ubuntu 14.04.5 LTS and TextAnalysis package, I'm receiving the following error with `Pkg.add("TextAnalysis")` 

```julia
$ sudo apt-get install julia
$ julia
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
   _ _   _| |_  __ _   |  Type "help()" to list help topics
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.2.1 (2014-02-11 06:30 UTC)
 _/ |\__'_|_|_|\__'_|  |  
|__/                   |  x86_64-linux-gnu

julia> Pkg.add("TextAnalysis")
INFO: Initializing package repository /home/ltan/.julia/v0.2
INFO: Cloning METADATA from git://github.com/JuliaLang/METADATA.jl
INFO: Cloning cache of BinDeps from git://github.com/JuliaLang/BinDeps.jl.git
INFO: Cloning cache of Blocks from git://github.com/JuliaParallel/Blocks.jl.git
INFO: Cloning cache of Compat from git://github.com/JuliaLang/Compat.jl.git
INFO: Cloning cache of DataArrays from git://github.com/JuliaStats/DataArrays.jl.git
INFO: Cloning cache of DataFrames from git://github.com/JuliaStats/DataFrames.jl.git
INFO: Updating cache of DataFrames...
INFO: Cloning cache of GZip from git://github.com/JuliaIO/GZip.jl.git
INFO: Cloning cache of Languages from git://github.com/johnmyleswhite/Languages.jl.git
INFO: Cloning cache of SHA from git://github.com/staticfloat/SHA.jl.git
INFO: Cloning cache of SortingAlgorithms from git://github.com/JuliaLang/SortingAlgorithms.jl.git
INFO: Cloning cache of StatsBase from git://github.com/JuliaStats/StatsBase.jl.git
INFO: Cloning cache of TextAnalysis from git://github.com/johnmyleswhite/TextAnalysis.jl.git
INFO: Cloning cache of URIParser from git://github.com/JuliaWeb/URIParser.jl.git
ERROR: Missing package versions (possible metadata misconfiguration):  DataFrames v(nothing,v"0.4.3") [84523937447b336dfcbf1747b123e9d1da1308c6[1:10]]

 in error at error.jl:21
 in resolve at pkg/entry.jl:350
 in resolve at pkg/entry.jl:316
 in edit at pkg/entry.jl:24
 in add at pkg/entry.jl:44
 in add at pkg/entry.jl:48
 in anonymous at pkg/dir.jl:28
 in cd at file.jl:22
 in cd at pkg/dir.jl:28
 in add at pkg.jl:19
```

**Comment by aviks** [comment]
Julia version `0.2.1` is ancient and certainly not supported any more. You will not be able to do much with that old a release. Please use the current released version `0.5`. 

The best way to get that version is to download the generic linux binaries from http://julialang.org/downloads/

**Comment by alvations** [comment]
Thanks for the note! I'm not sure how Julia package installation works but is there a way like how python `pip` checks for the requirements.txt before installation and throws an error that says that the Julia version is not high enough?
